client/state/selectors/get-jetpack-onboarding-progress.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { reduce, values } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { isJetpackOnboardingStepCompleted } from 'state/selectors';
+import { JETPACK_ONBOARDING_STEPS as STEPS } from 'jetpack-onboarding/constants';
+
+export default function getJetpackOnboardingProgress( state, siteId, steps ) {
+	if ( ! steps ) {
+		steps = values( STEPS );
+	}
```

Comment:
Hmm, not a big fan of this sort of fallbacks. I find they tend to make things less predictable. Can we either drop this, or fully embrace it (i.e. drop the `steps` arg to the selector)?